eng/tox/try_import.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# This script is used to execute bandit within a tox environment. Depending on which package is being executed against,
+# a failure may be suppressed.
+
+
+from subprocess import check_call, CalledProcessError
+import argparse
+import os
+import logging
+import sys
+import pdb
```

Comment:
```suggestion
import argparse
import logging
```

Reply:
Nit: you don't need all these imports. Otherwise this is perfect.

Reply:
Big oof. Totally agree. Cleaned up comment, applied `black`, didn't clean up imports. /facepalm